db/seeds/contributions/womanonrails.rb

```diff
@@ -40,6 +40,31 @@
 contribution.attributes = { locale: :pl, name: 'Naprawianie błędów' }
 contribution.save
 
+contribution = Contribution.new(params.merge(position: 5))
+contribution.attributes = { locale: :en, name: 'Automatic SMS notifications' }
+contribution.attributes = { locale: :pl, name: 'Automatyczne powiadomienia SMS' }
+contribution.save
+
+contribution = Contribution.new(params.merge(position: 6))
+contribution.attributes = { locale: :en, name: 'Phone number normalization (standard E.164)' }
+contribution.attributes = { locale: :pl, name: 'Normalizacja numerów telefonw (E.164)' }
+contribution.save
+
+contribution = Contribution.new(params.merge(position: 7))
+contribution.attributes = { locale: :en, name: 'RemoteLock sync (electronic locks)' }
```

Comment:
I did not create that feature :) Instead of that you can say that I did automation of account creation (from one place we create multiple account for users google account, fb, remote lock account)